network-api/networkapi/mozfest/templates/fragments/session_card.html

```diff
@@ -1,32 +1,21 @@
 {% load static wagtailcore_tags %}
-<a class="tw-block tw-group tw-no-underline tw-p-12 tw-border tw-border-gray-60/20 tw-text-[22px]"
-   href="{{ url }}">
-
-    <div class="tw-grid tw-grid-cols-1 medium:tw-grid-cols-5 tw-gap-8">
-
-        <div class="medium:tw-col-span-3">
-            <div class="tw-mb-8 tw-text-black">
-                <div class="tw-h3-heading tw-inline-block group-hover:tw-underline tw-font-zilla tw-transition tw-mb-6">{{ title }}</div>
-                {% if author_subheading %}
-                    <div class="tw-body-small">{{ author_subheading }}</div>
-                {% endif %}
-            </div>
-            <div class="tw-text-lg">
-                {{ text|richtext }}
-            </div>
-        </div>
-
-        <div class="medium:tw-order-2 tw-shrink-0 medium:tw-col-span-2">
-            <div class="tw-relative w-full medium:w-[250px] tw-h-[140px] tw-overflow-hidden">
-                {% if animated_thumbnail %}
-                    <img class="tw-absolute tw-top-1/2 tw--translate-y-1/2 tw-left-1/2 tw--translate-x-1/2 tw-transform tw-w-12 tw-h-12 tw-z-10 group-hover:tw-opacity-0 tw-transition tw-duration-500 tw-ease-in-out tw-delay-100"
-                         src="{% static '_images/mozfest/play-triangle.svg' %}" alt="">
-                    <video class="tw-absolute tw-object-cover tw-left-1/2 tw-top-1/2 tw-h-full tw--translate-y-1/2 tw--translate-x-1/2" src="{{ animated_thumbnail }}" autoplay loop muted width="500" height="200"></video>
-                {% endif %}
-                <img class="tw-w-full tw-object-cover tw-absolute tw-left-1/2 tw-top-1/2 tw-h-full tw--translate-y-1/2 tw--translate-x-1/2 {% if animated_thumbnail %}group-hover:tw-opacity-0 {% endif %} tw-transition tw-duration-500 tw-ease-in-out tw-delay-100"
-                     src="{{ image }}" alt="">
-            </div>
-        </div>
+<a class="tw-block tw-group tw-no-underline tw-p-6 tw-border tw-border-gray-60/20 tw-text-[22px] tw-h-full" href="{{ url }}">
+    <div class="tw-relative w-full medium:w-[250px] tw-h-[290px] tw-overflow-hidden tw-mb-8">
+        {% if animated_thumbnail %}
+            <img class="tw-absolute tw-top-1/2 tw--translate-y-1/2 tw-left-1/2 tw--translate-x-1/2 tw-transform tw-w-12 tw-h-12 tw-z-10 group-hover:tw-opacity-0 tw-transition tw-duration-500 tw-ease-in-out tw-delay-100"
+                    src="{% static '_images/mozfest/play-triangle.svg' %}" alt="">
+            <video class="tw-absolute tw-object-cover tw-left-1/2 tw-top-1/2 tw-h-full tw--translate-y-1/2 tw--translate-x-1/2" src="{{ animated_thumbnail }}" autoplay loop muted width="500" height="200"></video>
```

Comment:
Can we make this respect users motion preference? E.g. don't show animation if reduced motion is preferred? 

https://tailwindcss.com/docs/hover-focus-and-other-states#prefers-reduced-motion

Reply:
Added in https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11588/commits/231334cf73bed4c7a17f35f7b6d094f18a0db351